nextflow.config

```diff
@@ -303,11 +304,13 @@ manifest {
     defaultBranch   = 'master'
     nextflowVersion = '!>=24.04.2'
     version         = '1.2.0'
+
     doi             = '110.5281/zenodo.14916661'
 }
 
 // Nextflow plugins
 plugins {
+
     id 'nf-schema@2.3.0' // Validation of pipeline parameters and creation of an input channel from a sample sheet
```

Comment:
this should be 2.2.0 from the TEMPLATE.
2.2.1 should be acceptable as well I guess.
2.3.0 should err in latest-everything as far as I know